README.md

```diff
@@ -232,6 +232,24 @@ where:
 - `opts` *Object*:
   - `answers` *Array*: A collection of questions and their answers of form `[{ q: String, a: String }, ...]`
   - `comment` *String*: A text to be viewed by the Kaitiakis of a group
+  - `applicant` *Object*: Details about the applicant, in the form:
+
+    ```js
+    {
+      set: {
+        // required fields
+        preferredName: String,
+        legalName: String,
+        aliveInterval: EdtfIntervalString,
+        city: String,
+        country: String
+
+        // TODO: add optional fields
+      }
```

Comment:
I hadn't imagined these all being set at once. I think I imagined them as individual mutable fields.
I think it might be good to investigate that.

I wonder... If we can nest Strategies, like

```
props: {
  answers: Overwrite(),
  applicant: {
    preferredName: Overwrite()
  }
}
```

reason for doing this would be so applicant fields can be split up if it gets too big. 
I'm not sure crut would support what I sketched at the moment though

Lazy alternative - just put the applicant fields directly in the root object.

Reply:
Hmm, might have to go the lazy way, i checked and it doesnt support nested strategies :( 

```
- spec.props.applicant must be a valid strategy Error: Invalid strategy:
 - strategy must have a schema
 - strategy must have an isValid method
 - strategy must have a concat method
 - strategy must have a mapToOutput method
 - strategy must have an identity method
```

Reply:
When you say "directly in the root object", do you mean put it as a static prop?